pkg/ebpf/c/tracee.bpf.c

```diff
@@ -859,6 +859,10 @@ typedef struct config_entry {
     u8 events_to_submit[128]; // use 8*128 bits to describe up to 1024 events
 } config_entry_t;
 
+typedef struct netconfig_entry {
+    int capture_length; // amount of network packet payload to capture (pcap)
```

Comment:
I might need more fields here in near future, so I guessed having a map for the network config options would be ok.